pkgs/development/python-modules/lime/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+
+, matplotlib, numpy, scipy, tqdm, scikit-learn-extra, scikit-image
+
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "lime";
+  version = "0.2.0.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-dpYOTwVf61Pom1AiODuvyHtj8lusYmWYSwozPRpX94E=";
+  };
+
+  propagatedBuildInputs =  [
+    matplotlib
+    numpy
+    scipy
+    tqdm
+    scikit-learn-extra
```

Comment:
Do we need `scikit-learn-extra`?
It seems that lime itself depends on just `scikit-learn`.

Reply:
You are right, we don't need `scikit-learn-extra`. Fixed